integration_test/citus-pgbench-ddl.sql

```diff
@@ -1,8 +1,9 @@
+CREATE EXTENSION citus;
 ALTER TABLE pgbench_accounts DROP CONSTRAINT pgbench_accounts_pkey;
 ALTER TABLE pgbench_accounts ADD PRIMARY KEY (aid, bid);
 ALTER TABLE pgbench_tellers DROP CONSTRAINT pgbench_tellers_pkey;
 ALTER TABLE pgbench_tellers ADD PRIMARY KEY (tid, bid);
 SELECT create_distributed_table('pgbench_accounts', 'bid');
```

Comment:
Curious that these statements didn't fail, since that would have required the Citus extension to exist?

Reply:
Ah, I missed the part that explains this in the PR description. Nevermind :)

Reply:
Yeah it was basically https://youtu.be/aI0euMFAWF8?si=sHdBguJhj_aDwsVN